apps/studio/electron/main/run/terminal.ts

```diff
@@ -0,0 +1,116 @@
+import { MainChannels } from '@onlook/models/constants';
+import * as pty from 'node-pty';
```

Comment:
@abhiroopc84 this is the only place where node-pty is being used. We'd swap it out if necessary.